python/Ganga/testlib/GangaUnitTest.py

```diff
@@ -186,9 +214,7 @@ def stop_ganga():
     ShutdownManager._ganga_run_exitfuncs()
 
     # Undo any manual editing of the config and revert to defaults
-    from Ganga.Utility.Config import allConfigs
-    for package in allConfigs.values():
-        package.revertToDefaultOptions()
+    clear_config()
```

Comment:
I don't think I agree that this _should_ take time. If it is taking significant time then this is a bug, but either way I would argue it makes more sense to have in the Core shutdown as there's no reason not to clean up after these objects in the normal code as well